.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        python-version: "3.11"
+    - name: Install poetry
+      run: python3 -m pip install --user poetry==1.3.1
+    - name: Publish
+      env:
+        POETRY_HTTP_BASIC_PYPI_USERNAME: __token__
+        POETRY_HTTP_BASIC_PYPI_PASSWORD: ${{ secrets.PYPI_TOKEN }}
```

Comment:
I've set up an initial token in secrets for now. As soon as the first version is published, we can set up a new one scoped to just the project.

Reply:
Sounds good 👍